CMakeLists.txt

```diff
@@ -1,7 +1,24 @@
 cmake_minimum_required(VERSION 3.1)
 set(CMAKE_CXX_STANDARD 14)
 
-project(client)
+project(client LANGUAGES CXX VERSION 2.7)
```

Comment:
does that mean we need to maintain this number as well as in VERSION.cmake?

Reply:
good point I changed the version.cmake.
In the long run we might wan't do drop the MIRALL variables.
I hope that no script parse that cmake file to determine the version... @dschmidt ?

Reply:
actually ... probably yes. That was the reason to move everything to VERSION.cmake.